doc/upgrading.md

```diff
@@ -15,6 +15,15 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 * [Version 25](#version-25)
 * [Version 20](#version-20)
 
+## Version 31
+
+* [RMB-328](https://issues.folio.org/browse/RMB-328) Update to OpenJDK 11.
+  A few updates to `pom.xml` are necessary. For `maven-compiler-plugin`
+  update `version` to `3.8.1`. and use `<release>11</release>` instead
+  of `source` and `target` elements.
+  Update aspectj version to 1.9.5. Update `aspect-maven-plugin` with
+  groupId `com.nickwongdev` and version `1.12.6`.
```

Comment:
Can you add a link to an [example](https://github.com/folio-org/mod-configuration/pull/87/files)?

Reply:
Not examples yet, since no modules are yet merged.

Reply:
Doesn't the linked pull request qualify as an example?

Reply:
You want the upgrade manual to point to this huge PR itself ?

Reply:
Or do you mean this (shorter) one: https://github.com/folio-org/mod-inventory-storage/pull/485

Reply:
Yes, https://github.com/folio-org/mod-inventory-storage/pull/485/files is slightly more simple than https://github.com/folio-org/mod-configuration/pull/87/files I proposed above.